Tests/tests/test_TraceControl.cpp

```diff
@@ -0,0 +1,420 @@
+/**
+ * If not stated otherwise in this file or this component's LICENSE
+ * file the following copyright and licenses apply:
+ *
+ * Copyright 2022 RDK Management
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+#include "gtest/gtest.h"
+#include <gmock/gmock.h>
+
+#include "TraceControl.h"
+
+#include "FactoriesImplementation.h"
+#include "IarmBusMock.h"
```

Comment:
please remove unused things from the test, like iarm mock

Reply:
I will clean this up.
Thanks